src/main/java/uk/gov/hmcts/darts/audio/component/impl/OutboundFileProcessorImpl.java

```diff
@@ -90,13 +90,14 @@ public List<AudioFileInfo> processAudioForPlaybacks(Map<MediaEntity, Path> media
         List<AudioFileInfo> concatenatedAndMergedAudioFileInfos = new ArrayList<>();
         for (ChannelAudio audioFileInfoList :  concatenationsList) {
             AudioFileInfo mergedAudio = merge(audioFileInfoList.getAudioFiles());
+            OffsetDateTime concatStartTime = mergedAudio.getStartTime().atOffset(ZoneOffset.UTC);
+            OffsetDateTime concatEndTime = mergedAudio.getEndTime().atOffset(ZoneOffset.UTC);
             AudioFileInfo trimmedAudio = trimToPeriod(
                 mergedAudio,
-                mergedAudio.getStartTime().atOffset(ZoneOffset.UTC),
-                mergedAudio.getEndTime().atOffset(ZoneOffset.UTC)
+                concatStartTime.isAfter(mediaRequestStartTime) ? concatStartTime : mediaRequestStartTime,
```

Comment:
From what I've seen, it doesn't appear to make much difference - once we've chosen to trim to just use the media request startTime, endTime here?

Reply:
it depends if your media_request start/end times are different to your media start/end. also have to consider there may be multiple playback files generated due to gaps